vsutil.py

```diff
@@ -246,15 +246,16 @@ def depth(clip: vs.VideoNode,
                              dither_type=dither_type.value)
 
 
-def _readable_enums(enum: Type[Enum], module: str = 'vsutil') -> List[str]:
+def _readable_enums(enum: Type[Enum]) -> List[str]:
     """
     Returns a list of all possible values in `module.enum`.
     Extends the default `repr(enum.value)` behavior by prefixing the enum with the name of the module it belongs to.
     """
+    module = 'vapoursynth' if 'importlib' in enum.__module__ else enum.__module__
```

Comment:
I still think going with an optional module parameter to override is better, as it delegates the responsibility or removing the ambiguity (caused by the importlib._bootstrap value) to the caller, who has the required knowledge to properly resolve the knowledge gap. If an override is not provided, it should fall back to `enum.__module__`.

Really nitpicky, though.

Reply:
Accepted feedback in 8473e5dd4b1f8c5d0bade1a08e5ea03e94777dee.